benchmarks/benchmarks/linprog_benchmark_files/__init__.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from __future__ import division, print_function, absolute_import
+
+"""
+==============================================================================
+`` --  Problems for testing linear programming routines
+==============================================================================
+
+This module provides a comprehensive set of problems for benchmarking linear 
+programming routines, that is, scipy.optimize.linprog with method =
+'interior-point' or 'simplex'.
+
+"""
+
+"""
+All problems are from the Netlib LP Test Problem Set, courtesy of CUTEr
+ftp://ftp.numerical.rl.ac.uk/pub/cutest/netlib/netlib.html
+
+Converted from SIF (MPS) format by Matt Haberland
+"""
+
+#from .go_funcs_A import *
+#from .go_funcs_B import *
+#from .go_funcs_C import *
+#from .go_funcs_D import *
+#from .go_funcs_E import *
+#from .go_funcs_F import *
+#from .go_funcs_G import *
+#from .go_funcs_H import *
+#from .go_funcs_I import *
+#from .go_funcs_J import *
+#from .go_funcs_K import *
+#from .go_funcs_L import *
+#from .go_funcs_M import *
+#from .go_funcs_N import *
+#from .go_funcs_O import *
+#from .go_funcs_P import *
+#from .go_funcs_Q import *
+#from .go_funcs_R import *
+#from .go_funcs_S import *
+#from .go_funcs_T import *
+#from .go_funcs_U import *
+#from .go_funcs_V import *
+#from .go_funcs_W import *
+#from .go_funcs_X import *
+#from .go_funcs_Y import *
+#from .go_funcs_Z import *
```

Comment:
Removed. As you can tell, I didn't clean up after prototyping all this. Thanks for pointing everything out, though!